lib/rules/__tests__/index.test.mjs

```diff
@@ -35,13 +35,15 @@ describe('all rules', () => {
 describe('fixable rules', () => {
 	test.each(ruleNames)('"%s" should describe fixable in the documents', async (name) => {
 		const rule = await rules[name];
-
-		if (!rule.meta.fixable) return;
+		const cell = rule.meta.fixable ? '\\s+🔧\\s+' : ' ';
+		const wrapper = (x) => {
+			return rule.meta.fixable ? x : x.not;
+		};
 
 		const ruleDoc = await readFile(new URL(`../${name}/README.md`, import.meta.url), 'utf8');
 
-		expect(ruleDoc).toMatch('`fix` option');
-		expect(rulesListDoc).toMatch(new RegExp(`^.+\`${name}\`.+\\|.+\\|\\s+🔧\\s+\\|$`, 'm'));
+		wrapper(expect(ruleDoc)).toMatch('`fix` option');
```

Comment:
[suggest] How about just using `if` block and disabling the lint error like this, instead of a `wrapper` function? It seems a bit readable.

```js
if (rule.meta.fixable) {
	expect(ruleDoc).toMatch('`fix` option'); // eslint-disable-line jest/no-conditional-expect
}
```

Reply:
> It seems a bit readable.

You mean that it hurts the readability?
I was just trying to whip up a quick fix because the current tests let slip that typo.
I think that if we have to add a eslint-disable-line comment it's a sign that we are doing something wrong; **we should have two separate tests**.

Reply:
My previous example code was insufficient. The more accurate suggested code is below: ⬇️ 

```js
if (rule.meta.fixable) {
	expect(ruleDoc).toMatch('`fix` option');
} else {
	expect(ruleDoc).not.toMatch('`fix` option');
}
```

This does the same thing as `wrapper`, but more explicitly. I feel `wrapper` is a bit hacky.

> I think that if we have to add a eslint-disable-line comment it's a sign that we are doing something wrong

I don't think so for every `eslint-disable` comment. Linters may produce false positives (as Stylelint does), and sometimes, there are some cases that we can just ignore. 😅 (For example, `eslint-disable no-console` etc.)

I think the `wrapper` function just hides the conditional here and works around `jest/no-conditional-expect` issues.

Reply:
Lemme work on something gimme a minute.

Reply:
Tell me if 7ccbf91 is to your liking.

Reply:
Ah, the updated code uses the negative lookahead assertion (`(?!pattern)`); much better 👍🏼